vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -72,6 +74,10 @@ var (
 	username            = flag.String("username", "", "The username used in the prometheus server basic auth")
 	password            = flag.String("password", "", "The password used in the prometheus server basic auth")
 	memorySaver         = flag.Bool("memory-saver", false, `If true, only track pods which have an associated VPA`)
+	authHeader          = flag.Bool("auth-header", false, `If True will use header authentication, If true aslo authHeaderName and authHeaderValueName must be supplied`)
+	authHeaderName      = flag.String("auth-header-name", "", `What is the header name to use for header based Auth`)
```

Comment:
nit: "The header name for header based Auth". We usually prefer a statement instead of question for flag description

Reply:
done :)